.env.example

```
@@ -0,0 +1,48 @@
+GOOGLE_CLIENT_SECRET=your_google_client_secret
+
+# ── OAuth2 (Kakao) ────────────────────────────
+# 카카오 개발자센터에서 앱 키 발급
+# https://developers.kakao.com
+KAKAO_CLIENT_ID=your_kakao_client_id
+KAKAO_CLIENT_SECRET=your_kakao_client_secret
+
+# ── AWS S3 ────────────────────────────────────
+AWS_REGION=ap-northeast-2
+AWS_S3_BUCKET=your-bucket-name
+AWS_S3_BASE_URL=https://your-bucket-name.s3.ap-northeast-2.amazonaws.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_S3_BASE_URL key should go before the AWS_S3_BUCKET key

(UnorderedKey)